src/cdk/drag-drop/directives/drag.ts

```diff
@@ -139,6 +141,21 @@ export class CdkDrag<T = any> implements AfterViewInit, OnChanges, OnDestroy {
   /** Class to be added to the preview element. */
   @Input('cdkDragPreviewClass') previewClass: string | string[];
 
+  /**
+   *  Target for Drag Preview clone to append into.
+   *  `global` will place the clone at the end of `<body>`.
+   *  `parent` will place the clone in the parent to the `CdkDropList` .
+   *  `ElementRef` or `HTMLElement` will place the clone inside the element.
+   *
+   *  When `global` (default), the Drag Preview is appended to `<body>` to avoid
+   *  potential issues with `z-index` and `overflow: hidden`. By changing the
+   *  location you run risk of ancestors styled with `overflow: hidden` hiding part
+   *  or all of the Drag Preview, as well as other items styled with a higher
+   *  `z-index` overlapping the preview container.
+   */
+  @Input('cdkDragPreviewContainer') previewContainer:
+    'global' | 'parent' | ElementRef | HTMLElement;
```

Comment:
nit: the `ElementRef` should be `ElementRef<HTMLElement>`.

Reply:
done